run-platform.sh

```diff
@@ -222,6 +222,8 @@ build_services() {
     }
   elif [ "$first_setup" = true ] || [ "$opt_upgrade" = true ]; then
     echo -e "$blue_text""Pulling""$default_text"" docker images tag ""$blue_text""$opt_version""$default_text""."
+    # Try again on a slow network.
+    VERSION=$opt_version $docker_compose_cmd -f $script_dir/docker/docker-compose.yaml pull ||
     VERSION=$opt_version $docker_compose_cmd -f $script_dir/docker/docker-compose.yaml pull || {
```

Comment:
@hari-kuriakose a minor command I would add a TO-DO so that the retry will happen only if the pull fails.

Reply:
@ritwik-g Yes, it will retry only if the `docker compose pull` fails the first time.